sdk/core/azure-core-amqp/src/amqp/private/unique_handle.hpp

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <azure/core/internal/unique_handle.hpp>
+
+namespace Azure { namespace Core { namespace Amqp { namespace _detail {
+
+  template <typename T> struct UniqueHandleHelper;
+
+  template <typename T> using UniqueHandle = Core::_internal::UniqueHandle<T, UniqueHandleHelper>;
```

Comment:
I find it the opposite - convenient. @LarryOsterman, what is your opinion? You could break the tie.
It could be the `CoreAmqpUniqueHandle`, but I don't like it.